doc/command_ref.rst

```diff
@@ -141,7 +141,15 @@ Options
     Show the help.
 
 ``--installroot=<path>``
-    set install root
+    Specifies an alternative installroot, relative to where all packages will be
+    installed. Think of this like doing ``chroot <root> dnf`` except using
+    ``--installroot`` allows dnf to work before the chroot is created. The
+    following things are taken from or stored in installroot: cachedir, gpgkey,
```

Comment:
For Fedora repo with --installroot option - if in fedora.repo in installroot is gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-$releasever-$basearch
the gpg key will be importet from destination /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-22-x86_64, but gpg keys but will be stored in installroot <installroot>/etc/pki/rpm-gpg.